src/App.jsx

```diff
@@ -8,9 +8,9 @@ import InfoSidebar from "./components/InfoSidebar";
 import SignInPage from "./components/SignInPage";
 import SignUpPage from "./components/SignUpPage";
 import Profile from "./components/Profile";
+import Activity from "./components/Activity";
 
 import RecordActivity from "./components/RecordActivity";
-import Leaderboard from "./components/Leaderboard";
 import RewardsPage from "./components/RewardsPage";
```

Comment:
Could you add back in the Leaderboard component, and the /compete route? Github for some reason isn't merging it properly